.trunk/trunk.yaml

```diff
@@ -22,9 +22,10 @@ lint:
     - prettier@3.2.5
     - yamllint@1.34.0
   ignore:
-    - linters:
-        - yamllint
-        - prettier
+    - linters: [prettier]
+      paths:
+        - docs/docs/**
+    - linters: [yamllint, prettier]
```

Comment:
why not:

```yaml
- linters:
    - yamllint
    - prettier
```

Reply:
I think the [x, y, z] format is more readable in this case. Trunk also suggests that in their docs: https://docs.trunk.io/check/configuration/ignoring-issues#ignoring-multiple-files